packages/amplify-cli-core/API.md

```diff
@@ -1777,8 +1781,8 @@ type ProjectInfo = {
     existingProjectConfig?: $TSAny;
     existingTeamProviderInfo?: $TSAny;
     isNewProject?: boolean;
-    inputParams?: $TSAny;
-    localEnvInfo?: $TSAny;
+    inputParams: $TSObject;
```

Comment:
Why `$TSObject` vs `Record<string, unknown>`?

Reply:
inputParams is initialized as {} in several places and the doc in types.ts mentioned that this should be used for those initializers. It seemed like the right thing to do at the time